LOG.tbasicu

```diff
@@ -27,22 +27,23 @@ uses "File"
 #Region "Equates"
 
 ' Log equates
-
-%LT_Debug = 1 as Long 'Display only if isDebug = %TRUE
-%LT_Time = 2 as Long ' Prepend DateTime$
-%LT_ToConsole = 4 as Long
-%LT_ToConsoleTitle = 8 as Long
-%LT_ToMsgBox = 16 as Long
-%LT_ToLogFile= 32 as long
-%LT_ToStatusBar= 64 as long
-'%LT_ToRichedit = 128 as Long ?
+begin Const
```

Comment:
Hi Sebastian,

not a big deal, but just a tip - as the help file says, the %EQUATES are Long by default.

So - you are not needed to specify AS LONG.

Reply:
True, this is a side effect, at the very beginning, these were strings...